samples/AspNetIntegration/MultipleOutputBindings.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace AspNetIntegration
+{
+    //<docsnippet_aspnetcore_multiple_outputs>
+    public class MultipleOutputBindings
+    {
+        private readonly ILogger<MultipleOutputBindings> _logger;
+        public MultipleOutputBindings(ILogger<MultipleOutputBindings> logger)
+        {
+            _logger = logger;
+        }
+        [Function("MultipleOutputBindings")]
+        public MyOutputType Run([HttpTrigger(AuthorizationLevel.Function, "post")] HttpRequest req)
```

Comment:
Should we include both `get` and `post` ?

Reply:
discussed offline - original abstraction model samples also only have `post` and this is the most used case